pycbc/strain.py

```diff
@@ -377,6 +377,11 @@ def from_cli(opt, dyn_range_fac=1, precision='single',
                                           low_frequency_cutoff=lowfreq)
             strain = resample_to_delta_t(strain, 1.0/opt.sample_rate)
 
+        if opt.injection_file or opt.sgburst_injection_file \
+          or opt.ringdown_injection_file and not opt.channel_name:
```

Comment:
This line should be indented 8 more spaces than the previous, 4 more than the following.